v3/util/oid.go

```diff
@@ -24,6 +24,8 @@ import (
 
 var (
 	//extension OIDs
+	AdobeTimestampOID       = asn1.ObjectIdentifier{1, 2, 840, 113583, 1, 1, 9, 1}    // Adobe Timestamp x509 extension
```

Comment:
Is the correct formatting "timestamp" or "time-stamp".  RFC3161 has it as "Time-stamp" and most online references I can find follow this trend although I've not found any record of Adobe actually defining what their OID should be called.

Reply:
addressed